tests/DontTestAsset/NotSerialisable.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DontTestAsset;
+
+use Dont\DontSerialise;
+
+final class NotSerialisable
```

Comment:
Yes, we just went with the brit approach: 
<img width="692" alt="bd4567" src="https://cloud.githubusercontent.com/assets/154256/19267855/59daae8a-8fb1-11e6-91c2-2d2800436832.png">